docs/styles/typography.md

```diff
@@ -1,47 +1,77 @@
-Base: f = font-size
-
----
-
-### Modifiers:
-
-- 1 = 1st step in size scale
-- 2 = 2nd step in size scale
-- 3 = 3rd step in size scale
-- 4 = 4th step in size scale
-- 5 = 5th step in size scale
-- 6 = 6th step in size scale
-- 7 = 7th step in size scale
-
-### Font Sizes:
-
-```sh
-.f1 { font-size: 3rem; }
-.f2 { font-size: 2.25rem; }
-.f3 { font-size: 1.5rem; }
-.f4 { font-size: 1.25rem; }
-.f5 { font-size: 1rem; }
-.f6 { font-size: 0.875rem; }
-.f7 { font-size: 0.75rem; }
+<div class="pb7"></div>
+<div class="f2 c-on-base pv5">Typography</div>
+Typography is used to communicate informations the most efficient way possible through legibility and visual hierarchy. It's a crucial tool to guide users on their tasks. It should be used on clear and delightful way. For that, it's important to define different **typography styles** for an app by setting a combination of different typography attributes: **Typeface, weight, size, capitalization and letter spacing.**
+
+Typography styles work as single-purpose CSS classes used to modify the typography configuration of an app. Each class configures the following CSS attributes: ** font-family, font-size, font-weight, text-transform and letter-spacing. **
+
+<!-- Headings -->
+<div class="flex mt7 mb3 items-center">
```

Comment:
Isn't `tc` (text-center) enough in this case?

Reply:
Fixed.